### ISSUE TITLE
I found an app that is suspiciously close to ish 

### ISSUE THREAD

**Issue by Wh0ba** [tbodt/ish#371]
So I searched for Termius on the AppStore to use ssh 
And an app showed up called iShell
Which claims to be a portable Linux 
It uses Alpine Linux just like ish
It has an improved UI
The settings look identical (they added more color themes)

Here is the link for the app 
https://itunes.apple.com/us/app/ishell/id1455946930?mt=8

Is that legal and does it respect your licensing rights?

**Comment by ecstaticax** [comment]
I confirm that it is iSH with some polishing on the interface.

**Comment by saagarjha** [comment]
@tbodt has already gotten in touch with Apple to clarify the licensing situation with the developer of that app.

**Comment by tbodt** [comment]
It's not legal, if the developer doesn't respond to my emails I will ask for apple to take it down.

**Comment by ecstaticax** [comment]
At least now you now that apple can approve your app. Why don’t you put on AppStore legally? Please, don’t ask for subscription. I will happily pay to buy it. It’s a wonderful thing.

**Comment by jusdepatate** [comment]
> 
> 
> At least now you now that apple can approve your app. Why don’t you put on AppStore legally? Please, don’t ask for subscription. I will happily pay to buy it. It’s a wonderful thing.

Why "legally" ? TestFlight isn't illegal
And @tbodt waits to get a 100% stable version of iSH before public release

**Comment by ecstaticax** [comment]
Yes, it’s legal, but on AppStore I think it would be better.

**Comment by saagarjha** [comment]
> At least now you now that apple can approve your app.

Obligatory reminder that the App Store review process is not necessarily consistent or even objective.

**Comment by ecstaticax** [comment]
I agree

**Comment by tbodt** [comment]
I've obtained a decrypted version of the app and it includes plenty of code for showing ads that can be activated remotely at any time. sigh

**Comment by ecstaticax** [comment]
I think you can ask Apple to remove it

**Comment by tbodt** [comment]
Apple has informed me that the app has been removed. Seems they wait 30 days for a response. Smart move, considering a lawsuit could never happen that quickly.

**Comment by mirorauhala** [comment]
I don’t really understand how the copy was illegal. Yes it was a fork of this repository. Repository which is licensed under an open-source, copyleft license. What I cannot wrap my head around is that, first you make code, license it with a free license, and then complain that someone used in within their rights. Rights that you gave them.

If you do not want others to publish the same app with or without modification, you should reconsider the licensing of the project.

Also note that if you notified Apple that the other is a copy of yours. They may have deleted it on the grounds of being GPLv3 licensed as I discussed here #34. 

TL;DR: There’s no such thing as an illegal copy of open-source licensed software. That’s why they are open-source.

**Comment by tbodt** [comment]
You're free to make your own modified version of iSH and put it on the app store, as long as you release your code, which didn't happen in this case.
